scipy/sparse/linalg/_isolve/iterative.py

```diff
@@ -27,40 +27,46 @@ def bicg(A, b, x0=None, *, rtol=1e-5, atol=0., maxiter=None, M=None, callback=No
     Parameters
     ----------
     A : {sparse matrix, ndarray, LinearOperator}
-        The real or complex N-by-N matrix of the linear system.
+        Real or complex N-by-N matrix of the linear system.
         Alternatively, ``A`` can be a linear operator which can
         produce ``Ax`` and ``A^T x`` using, e.g.,
         ``scipy.sparse.linalg.LinearOperator``.
     b : ndarray
-        Right hand side of the linear system. Has shape (N,) or (N,1).
-    x0 : ndarray
-        Starting guess for the solution.
+        Right hand side of the linear system with shape (N,) or (N,1).
+    x0 : ndarray, optional, default: None
+        Starting guess for the solution with shape (N,) or (N,1).
     rtol, atol : float, optional
         Parameters for the convergence test. For convergence,
         ``norm(b - A @ x) <= max(rtol*norm(b), atol)`` should be satisfied.
         The default is ``atol=0.`` and ``rtol=1e-5``.
-    maxiter : integer
+    maxiter : integer, optional, default: None
         Maximum number of iterations.  Iteration will stop after maxiter
         steps even if the specified tolerance has not been achieved.
-    M : {sparse matrix, ndarray, LinearOperator}
+    M : {sparse matrix, ndarray, LinearOperator}, optional, default: None
         Preconditioner for A.  The preconditioner should approximate the
         inverse of A.  Effective preconditioning dramatically improves the
         rate of convergence, which implies that fewer iterations are needed
         to reach a given error tolerance.
-    callback : function
+    callback : function, optional, default: None
```

Comment:
Here and above where the default is None I would prefer to omit the added text

Reply:
done